next.clayui.com/content/docs/css/components/loading-indicator.md

```diff
@@ -0,0 +1,156 @@
+---
+title: "Loading Indicator"
+description: "The loading indicator shows the user that an external process, like a connection, is being executed."
+---
+
+<div class="nav-toc">
+
+- [Loading Icon Dotted](#loading-icon-dotted)
+- [Loading Icon Dotted Light](#loading-icon-dotted-light)
+- [Button with Loading Icon Dotted](#button-with-loading-icon-dotted)
+
+</div>
+
+<div class="clay-site-alert alert alert-info">
+	Check the <a href="https://liferay.design/lexicon">Lexicon</a> <a href="https://liferay.design/lexicon/core-components/loading-indicator/">Loading Indicator Pattern</a> for a more in-depth look at the motivations and proper usage of this component.
+</div>
+
+<div class="clay-site-alert alert alert-warning">
+	See the implementation of the <a href="/docs/components/loading-indicator.html">Loading Indicator component in React</a> following the Lexicon <a href="https://liferay.design/lexicon/core-components/loading-indicator/">Loading Indicator Pattern</a>.
+</div>
+
+> These animations are made using only CSS.
```

Comment:
We can remove this text from the blockquote and add it as a warning or info. Since it is more an informative phrase.

Reply:
Changed to info alert